www/js/controller/placeDetails.controller.js

```diff
@@ -2,8 +2,16 @@ angular.module('vamosJuntas').controller('placeDetailsController', ['$scope', 'p
 
   $scope.placeDetails = placeFactory.getPlace();
 
+  $scope.getTotalOfOccurrences = function(place) {
```

Comment:
I think we could create a factory or service to put this `getTotalOfOccurrences` and avoid duplicated code.